### PR TITLE
enh(AI): More additions to AI admin docs

### PR DESCRIPTION
**Pull request by marcelklehr**

### CODE REVIEW

File: admin_manual/ai/app_assistant.rst [hunk 1]
```diff
@@ -63,6 +63,20 @@ In order to make use of text processing features in the assistant, you will need
...
+
+* *Generate text* (Tested with OpenAI GPT-3.5 and Llama 3.1 8B)
+* *Summarize* (Tested with OpenAI GPT-3.5 and Llama 3.1 8B)
+* *Generate headline* (Tested with OpenAI GPT-3.5 and Llama 3.1 8B)
+* *Extract topics* (Tested with OpenAI GPT-3.5 and Llama 3.1 8B)
+
+Additionally, *integration_openai* also implements the following Assistant Tasks:
+
+* *Context write* (Tested with OpenAI GPT-3.5)
+* *Reformulate text* (Tested with OpenAI GPT-3.5)
```
Comment (julien-nc) [inline admin_manual/ai/app_assistant.rst:76]: It implements more task types:
* Translate
* Generate image
* Audio to text

**Review by julien-nc — APPROVED** [review]

**Review by DaphneMuller — APPROVED** [review]
looks good to me, thank you Marcel!

File: admin_manual/ai/app_assistant.rst (hunk 1 above)
Comment (marcelklehr) [reply admin_manual/ai/app_assistant.rst:76]: But this is the Text processing section

**Pull request merged**